app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -1,26 +1,87 @@
 package com.intive.picover.profile.view
 
 import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.Image
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.material3.Button
+import androidx.compose.material3.Divider
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
 import com.intive.picover.R
 import com.intive.picover.profile.viewmodel.ProfileViewModel
+import com.skydoves.landscapist.ImageOptions
+import com.skydoves.landscapist.coil.CoilImage
 
 @Composable
 fun ProfileScreen(viewModel: ProfileViewModel) {
+	val profile by viewModel.profile.collectAsState()
 	val context = LocalContext.current
 	Column(
-		modifier = Modifier.fillMaxSize(),
+		modifier = Modifier
+			.verticalScroll(rememberScrollState())
+			.fillMaxSize()
+			.padding(top = 10.dp),
 		horizontalAlignment = Alignment.CenterHorizontally,
 	) {
+		profile.let {
```

Comment:
i think we can drop let here and just use profile.photo etc in the implementation of layout :) for me it is more readable to have this in this way :)

Reply:
I'm not attached to it so I will change it :D

Reply:
Done